streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -189,5 +190,21 @@ private[ui] class StreamingPage(parent: StreamingTab)
     }
     UIUtils.listingTable(headers, generateDataRow, data, fixedWidth = true)
   }
+
+  private def generateBatchListTables(): Seq[Node] = {
+    val runningBatches = listener.runningBatches.sortBy(_.batchTime.milliseconds).reverse
+    val waitingBatches = listener.waitingBatches.sortBy(_.batchTime.milliseconds).reverse
+    var content =
```

Comment:
This is a little hard to read. Please rearrange the code a little bit. Also better to not use var. Rather 
```
val activeBatchesContent = {
    <h4 id="active">Active Batches ({runningBatches.size + waitingBatches.size})</h4> ++
       new ActiveBatchTable(runningBatches, waitingBatches).toNodeSeq
}

val completedBatchesContent = {
...
}

...
activeBatchesContent ++ completedBatchesContent
```